src/lib.rs

```diff
@@ -137,6 +138,36 @@ impl<T> AtomiQueue<T> {
         Ok(Some(value))
     }
 
+    /// Clones the top value off the back end of the queue.
```

Comment:
This should probably be phrased as `Clones the most recently pushed value`

Reply:
Changed